From d1ec2a081b1b326672e2635d1ec45dd9aa9c84c8 Mon Sep 17 00:00:00 2001
From: Paul Barker <paul.barker@sancloud.com>
Date: Thu, 17 Jun 2021 10:33:30 +0100
Subject: [PATCH 4/4] MAINTAINERS: Adopt SanCloud dts files as supported

Myself and Marc are happy to maintain these device tree files going
forward.

Signed-off-by: Paul Barker <paul.barker@sancloud.com>
Acked-by: Marc Murphy <marc.murphy@sancloud.com>
---
 MAINTAINERS | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 8c5ee008301a..28826d26e9cc 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -16234,6 +16234,12 @@ F:	drivers/phy/samsung/phy-s5pv210-usb2.c
 F:	drivers/phy/samsung/phy-samsung-usb2.c
 F:	drivers/phy/samsung/phy-samsung-usb2.h
 
+SANCLOUD BEAGLEBONE ENHANCED DEVICE TREE
+M:	Paul Barker <paul.barker@sancloud.com>
+R:	Marc Murphy <marc.murphy@sancloud.com>
+S:	Supported
+F:	arch/arm/boot/dts/am335x-sancloud*
+
 SC1200 WDT DRIVER
 M:	Zwane Mwaikambo <zwanem@gmail.com>
 S:	Maintained
-- 
2.30.2

